homework_16/employee.py

```diff
@@ -0,0 +1,34 @@
+from abc import ABC
+
+
+class Employee(ABC):
+
+    name: str
```

Comment:
Attributes like this one is better to create with private access and use getter and setter methods for acting with them.